### ISSUE TITLE
[donation_receipt] shortcode doesn't work?

### ISSUE THREAD

**Issue by qriouslad** [Charitable/Charitable#334]
Hi Eric,

I followed instructions at https://www.wpcharitable.com/documentation/customizing-the-donation-receipt-page/, but upon publishing the page (step 2), it takes me to a white page. 

WP debug log churns out the following:

```
Uncaught Error: Call to undefined function charitable_template_donation_receipt_output() 
in /var/www/mysite.com/htdocs/wp-content/plugins/charitable/includes/shortcodes/
class-charitable-donation-receipt-shortcode.php:34

Type: PHP Fatal error
File: /var/www/mysite.com/htdocs/wp-content/plugins/charitable/includes/shortcodes/
class-charitable-donation-receipt-shortcode.php
```

Any clue?

**Comment by qriouslad** [comment]
Note: however, if I login using the account creating that page, and then make a donation to a campaign, the receipt page shows up just fine with info from summary.php and details.php of the receipt template folder. All the while, edit screen of the receipt page is still a blank white page.

**Comment by ericnicolaas** [comment]
Hi Bowo! 

Sorry for taking a while to get back to you on this. Are you using this alongside a page builder? The problem might be because the shortcode is not loaded in the admin. I've seen this issue crop up with Divi in the past, though I thought I'd patched that issue.

**Comment by qriouslad** [comment]
Hi Eric!

Thanks for the reply. Yes, I'm using Extra theme, which integrates with the Divi builder. However, after some UX considerations, I actually decided to just 'bypass' displaying the donation receipt page and redirect users to a 3rd party payment gateway I'm using for this project: https://www.xendit.co (payment via their invoice UI). 

So, all's good for now, but let me know if you want me to test further and extract some insights for you to troubleshoot the issue. I'm loving Charitable by the way, so nice to be able to override the templates easily. :-)

**Comment by ericnicolaas** [comment]
Thanks for the kind words @qriouslad :)

I'll have to check out Xendit. Looks like a nice option for our customers in Southeast Asia. 

By the way, this issue will be resolved in Charitable 1.5. Found the issue for it: #316

**Comment by qriouslad** [comment]
Would be awesome if you could build a Xendit addon for Charitable @ericnicolaas 

Thanks for including the bugfix for 1.5. ;-)
